assets/aml-benchmark/components/src/utils/aml_run_utils.py

```diff
@@ -0,0 +1,96 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+"""Helper functions for AML runs."""
+from typing import List, Optional, Tuple, cast
+import os
+import tempfile
+import re
+
+import mlflow
+from mlflow.entities import Run as MLFlowRun
+from azureml.core import Run
+from utils.logging import get_logger
+
+
+logger = get_logger(__name__)
+
+
+def get_experiment_name() -> str:
+    """Get the current experiment name."""
+    return Run.get_context().experiment.name
+
+
+def get_parent_run_id() -> str:
+    """Get the run id of the parent of the current run."""
+    return cast(str, Run.get_context().parent.id)
+
+
+def get_all_runs_in_current_experiment() -> List[MLFlowRun]:
```

Comment:
Is there a reason to sometimes use AML SDK and sometimes mlflow? 

For example, this code is simpler in AML SDK (if you want AML runs in the output, of course)

```python
experiment = Experiment(workspace=workspace, name=experiment_name)
runs = experiment.get_runs()
```

Reply:
There were some properties I think that I did not find in AML SDK. So, I used mlflow to get them.